apps/dokploy/templates/budibase/index.ts

```diff
@@ -0,0 +1,60 @@
+import {
+	type DomainSchema,
+	type Schema,
+	type Template,
+	generateRandomDomain,
+} from "../utils";
+
+export function generate(schema: Schema): Template {
+	const mainDomain = generateRandomDomain(schema);
+
+	const apiKey = Array.from({ length: 32 }, () =>
+		Math.floor(Math.random() * 16).toString(16),
+	).join("");
+	const encryptionKey = Array.from({ length: 32 }, () =>
+		Math.floor(Math.random() * 16).toString(16),
+	).join("");
+	const jwtSecret = Array.from({ length: 32 }, () =>
+		Math.floor(Math.random() * 16).toString(16),
+	).join("");
+	const couchDbPassword = Array.from({ length: 32 }, () =>
+		Math.floor(Math.random() * 16).toString(16),
+	).join("");
+	const redisPassword = Array.from({ length: 32 }, () =>
+		Math.floor(Math.random() * 16).toString(16),
+	).join("");
+	const minioAccessKey = Array.from({ length: 32 }, () =>
+		Math.floor(Math.random() * 16).toString(16),
+	).join("");
+	const minioSecretKey = Array.from({ length: 32 }, () =>
+		Math.floor(Math.random() * 16).toString(16),
+	).join("");
+	const watchtowerPassword = Array.from({ length: 32 }, () =>
+		Math.floor(Math.random() * 16).toString(16),
+	).join("");
```

Comment:
use the generatePassword FN from utils directly in `apps/dokploy/templates/utils/index.ts`